tasks/build.py

```diff
@@ -324,28 +349,40 @@ def apply_cvmfs_customizations(cvmfs_customizations, arch_job_dir):
             #      for now, only existing mappings may be customized
 
 
-def prepare_jobs(pr, event_dir, build_env_cfg, arch_map, repocfg):
+def prepare_jobs(pr, cfg, event_info, action_filter):
     """prepare job directory with pull request and cfg/job.cfg as well as
        additional config files
 
     Args:
-        pr: github.PullRequest.Pullrequest object
-        event_dir: base directory for all jobs created for this event
-        build_env_cfg: build env configuration
-        arch_map: maps OS/SW_DIR to Slurm parameters
-        repocfg: repository config
+        pr (github.PullRequest.Pullrequest): object to interact with pull request
+        cfg (dict): dictionary holding full configuration (by default defined in app.cfg)
+        event_info (dict): event received by event_handler
+        action_filter (EESSIBotActionFilter): used to filter which jobs shall be prepared
 
     Returns:
         jobs: list of the created jobs
     """
     fn = sys._getframe().f_code.co_name
 
+    app_name = cfg[GITHUB].get(APP_NAME)
+    build_env_cfg = get_build_env_cfg(cfg)
+    arch_map = get_architecturetargets(cfg)
+    repocfg = get_repo_cfg(cfg)
+
     base_repo_name = pr.base.repo.full_name
     log(f"{fn}(): pr.base.repo.full_name '{base_repo_name}'")
 
     base_branch_name = pr.base.ref
     log(f"{fn}(): pr.base.repo.ref '{base_branch_name}'")
 
+    # create run dir (base directory for potentially several jobs)
+    # TODO may still be too early (before we get to any actual job being
+    #      prepared below when calling 'download_pr')
+    # instead of using a run_dir, maybe just create a unique dir for each
+    # job to be submitted? thus we could easily postpone the create_pr_dir
+    # call to just before download_pr
+    ym, pr_id, run_dir = create_pr_dir(pr, cfg, event_info)
```

Comment:
no idea what `ym` is, deserves better variable name

Reply:
Replaced everywhere with `year_month`